src/extensions/electrodegeometry/ElectrodeGeometryWidget/ElectrodeGeometryWidget.tsx

```diff
@@ -175,6 +182,10 @@ const onUpdateLayerProps = (layer: CanvasWidgetLayer<ElectrodeLayerProps, Electr
 const paintElectrodeGeometryLayer = (painter: CanvasPainter, props: ElectrodeLayerProps, state: ElectrodeLayerState) => {
     painter.wipe()
     const useLabels = state.pixelRadius > 5
+    // The following three lines are visualizations to confirm the scaled image remains centered.
+    // painter.fillWholeCanvas('rgb(224, 196, 224)')
+    // const electrodesBoundngBox = getElectrodesBoundingBox(props.electrodes, 0)
+    // painter.drawRect(electrodesBoundngBox, {color: 'black', width: 4})
```

Comment:
These lines collectively paint the whole canvas a light pink and then draw a black box around the centers of the electrodes (exclusive of the radii). Just for visualization purposes.